app/src/main/java/co/stonephone/stonecamera/StoneCameraApp.kt

```diff
@@ -143,8 +150,13 @@ fun StoneCameraApp(
                 .align(Alignment.BottomCenter),
             verticalArrangement = Arrangement.SpaceBetween
         ) {
-            plugins.map {
-                it.renderTray(stoneCameraViewModel, it)
+
+            Box(
+                modifier = Modifier.fillMaxWidth(),
+            ) {
```

Comment:
These previous 3 lines look unnecessarily spread out; is there a reason why they're not on the same line?